Libraries/Components/Slider/Slider.js

```diff
@@ -127,6 +127,26 @@ var Slider = React.createClass({
      * Used to locate this view in UI automation tests.
      */
     testID: PropTypes.string,
+
+    /**
+     * The color used for the progress
+     * @platform android >=21
```

Comment:
I'm not sure how this will render on the website. Can you try using `cd website; npm install; npm start` and post a screenshot please? It might not render well, in that case it's fine to explain this in the doc block above.